index.html

```diff
@@ -0,0 +1,177 @@
+          <p>
+          Listen to speakers from renowned influentials 
+          youths of our time as they share on how
+          youth can win.
+          </p>
+        </div>
+      </div>
+      <div class="card_container">
+        <div class="card_img">
+          <img  src="./assets/exhibition_icon.png" alt="exhibition"> 
+        </div>
+        <div class="card_title">
+          <h3>Exhibition</h3>
+        </div>
+        <div class="card_description">
+          <p>Let's meet the creations of enterpreneurs from various fields as they share
+            their products.
+          </p>
+        </div>
+      </div>
+      <div class="card_container">
+        <div class="card_img">
+          <img src="./assets/forum_icon.png" alt="forum">
+        </div>
+        <div class="card_title">
+          <h3>Forum</h3>
+        </div>
+        <div class="card_description">
+          <p> We have time to share our thoughts and opinions with experts by topic.</p>
+        </div>
+      </div>
+      <div class="card_container">
+        <div class="card_img">
+          <img src="./assets/workshop_icon.png" alt="workshop"> 
+        </div>
+        <div class="card_title">
+          <h3>workshop</h3>
+        </div>
+        <div class="card_description">
+          <p>
+          You can create your own creations using open source instead of 
+          just looking at them.
+          </p>
+        </div>
+      </div>
+      <div class="card_container">
+        <div class="card_img">
+          <img  src="./assets/party_icon.png" alt="party_icon">
+        </div>
+        <div class="card_title">
+          <h3>Party</h3>
+        </div>
+        <div class="card_description">
+          <p>All work no play...On Sunday it's about responsible partying,
+            while creating networks</p>
+        </div>
+      </div>
+      </div>
+      <button class="join_button"><h3>JOIN KENYA YOUTH SUMMIT 2022 </h3></button>
+      <div class="see_prog">
+        <span>See the whole Program</span>
+      </div>
+    </section>
+    <section id="main_speakers" class="main_speakers">
+      <h2 class="speakers_headline">Main Speakers</h2>
+      <div id="speakers_wrap" class="speakers_wrap">
+    </div>
+      <button  class="more_button" type="button">MORE
+        <img src="./assets/arrow_down.png" alt="more">
+      </button>
+    </section>
+  </main>
+  <footer>
+    <div class="sponsor_headline">
+      <h2>Sponsors</h2>
+    </div>
+    <div class="sponsors">
+      <li><a href="#">Google</a></li>
+      <li><a href="#">Mozilla</a></li>
+      <li><a href="#">AirBnB</a></li>
+      <li><a href="#">Kakao</a></li>
+      <li><a href="#">Naver</a></li>
+    </div>
+    <div class="attribute">
+      <div class="attribute_img">
+        <img src="./assets/footer_image.png" alt="footerimg">
+      </div>
+      <p>Original work is from Cindy Shin,<br>
+         <span> Creative Commons Some Rights Reserved </span></p>
+    </div>
+  </footer>
```

Comment:
in the mobile version, the footer section of the page does not display, i suggest you fix the issue in the script.js file that causes this error to improve your design

Reply:
- [ ] I see the requested changes but have tested it in my machine and I don't see that overflow.
- [ ] The footer is not supposed to display in the mobile version,That is according to the mobile version template provided.
- [ ] From my dashboard I see the project has been approved so I will go ahead and merge and submit